### PR TITLE
Adding rmats2sashimiplot recipe

### PR DESCRIPTION
**Pull request by matthieujung**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

I'm sorry but I encounter some difficulties to run simulate-travis.py script. Can anyone checks this recipe before merging with the original repository?
Thanks in advance.

### CODE REVIEW

File: recipes/rmats2sashimiplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package:
+  name: rmats2sashimiplot
+  version: "2.0.0"
+
+source:
+  git_rev: c3690679de217238b47251c574b3ec2b89af5745
```
Comment (jenzopr) [inline recipes/rmats2sashimiplot/meta.yaml]: Consider using tagged releases that are accessible as tarballs from the “releases” section of the github repo.
Also don't forget to add a hash, like `md5` or `sha256`.

File: recipes/rmats2sashimiplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - python ==2.6*|2.7*
+    - matplotlib
+    - scipy
+    - pysam ==0.9.0
+    - samtools ==1.3.1
+
+build:
+  noarch: python
+  preserve_egg_dir: True
+
```
Comment (jenzopr) [inline recipes/rmats2sashimiplot/meta.yaml]: Add a tests section, e.g. with
```
test:
  commands:
    - rmats2sashimiplot -h 
```

File: recipes/rmats2sashimiplot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+about:
+  home: https://github.com/Xinglab/rmats2sashimiplot
```
Comment (jenzopr) [inline recipes/rmats2sashimiplot/meta.yaml]: Add a `license` directive (in this case should be `license: GNU GPLv2`, as well as a `license_file`, which is available as `license_file: LICENSE`.
Also add a short summary under the `summary` directive.

File: recipes/rmats2sashimiplot/build.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (jenzopr) [inline recipes/rmats2sashimiplot/build.sh]: Consider adding a proper shebang line

File: recipes/rmats2sashimiplot/meta.yaml [hunk 5]
```diff
@@ -20,5 +20,16 @@ build:
   noarch: python
```
Comment (jenzopr) [inline recipes/rmats2sashimiplot/meta.yaml]: The current build failed for python3. Can you add a `skip: True # [not py27]`? Subsequently, `python ==2.6*|2.7*` can be replaced by `python`.

File: recipes/rmats2sashimiplot/meta.yaml [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
+  version: "2.0.0" # Version number indicated in the setup.py file.
 
 source:
-  git_rev: c3690679de217238b47251c574b3ec2b89af5745
-  git_url: https://github.com/Xinglab/rmats2sashimiplot.git
+  url: https://github.com/Xinglab/rmats2sashimiplot/archive/v0.0.1.tar.gz
+  md5: fe92f2dcaceacc18ed864e269f866766
 
 requirements:
   build:
```
Comment (jenzopr) [inline recipes/rmats2sashimiplot/meta.yaml]: `setuptools` is a missing requirement for the build

File: recipes/rmats2sashimiplot/meta.yaml [hunk 7]
```diff
@@ -20,5 +20,16 @@ build:
   noarch: python
   preserve_egg_dir: True
 
+test:
+  commands:
+    - rmats2sashimiplot -h
+  imports:
+    - pysam
```
Comment (jenzopr) [inline recipes/rmats2sashimiplot/meta.yaml]: The tests for required packages are not needed, they're handled within the respective package

**Comment by matthieujung** [comment]
Thanks a lot @jenzopr for your help.

**Pull request merged**